### PR TITLE
[REF] web: convert pie_chart widget to owl

### PR DESCRIPTION
**Pull request by FrancoisGe**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/tests/views/kanban_view_tests.js [hunk 1]
```diff
@@ -10082,21 +10041,14 @@ QUnit.module("Views", (hooks) => {
     });
 
     QUnit.test("kanban widget supports options parameters", async (assert) => {
```
Comment (aab-odoo) [inline addons/web/static/tests/views/kanban_view_tests.js:10043]: suggestion: "kanban widget can extract props from attrs"

**Review by aab-odoo — APPROVED** [review]

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff r+

**Comment by Polymorphe57** [comment]
robodoo r+

**Comment by Polymorphe57** [comment]
robodoo r-

**Comment by Polymorphe57** [comment]
On y est!

**Comment by Polymorphe57** [comment]
robodoo r+

**Comment by Polymorphe57** [comment]
robodoo r+

**Comment by Polymorphe57** [comment]
robodoo retry

**Pull request closed without merging**